platform/platform-app/src/main/resources/features.xml

```diff
@@ -528,6 +528,11 @@
     </feature>
     <!-- End CXF Features -->
 
+    <feature name ="platform-configuration-listener" install="manual" version="${project.version}"
```

Comment:
Is there a reason why this was not just added to the platform-app feature instead of adding it to org.apache.karaf.features.cfg?